src/api/entities/Asset/__tests__/Settlements.ts

```diff
@@ -236,17 +236,17 @@ describe('Settlements class', () => {
     });
 
     beforeEach(() => {
-      fromPortfolio = entityMockUtils.getDefaultPortfolioInstance({
```

Comment:
It doesn't make sense to me to refactor these into a shared method with those from the `canSettle` case, but by changing the order they shouldn't be the same block as above from the perspective of static analysis.